test/internal/helpers/Player.java

```diff
@@ -846,6 +846,17 @@ public static Cleanups withRange() {
         });
   }
 
+  /**
+   * Does nothing, but ensures the given property is reverted as part of cleanup
+   *
+   * @param key Key of property
+   * @return Restores the previous value of the property
+   */
+  public static Cleanups withProperty(final String key) {
```

Comment:
I like this. I have all sorts of tests that call methods which set a property as a side effect.
Could list the properties I expect to be modified, rather than cleaning them up in an afterEach method, say.

Reply:
Yeah. We can't necessarily judge every changed pref but I think this should be the way forwards